src/shared/constants.ts

```diff
@@ -204,3 +204,5 @@ export const defaultLaunchJson = JSON.stringify(
 	},
 	undefined, "\t"
 );
+
+export const LATEST_SDK_VERSION = "latest";
```

Comment:
It's probably worth a comment on this explaining why it's used (and what the general behaviour should be when it's used).

We could also name this something like 'MAX_VERSION' and set it to something high like `999.999` to avoid having to special-case in `versionIsAtLeast`. I'm not certain if that's better, but it could avoid accidentally trying to treat the string as a version number elsewhere that could fail.

Reply:
`MAX_VERSION` is fine with me. I initially wanted something that would be readable for display but just changed to checking for matching to `MAX_VERSION` instead.